Sentiment_analysis/Sentiment_analysis/settings.py

```diff
@@ -0,0 +1,122 @@
+"""
+Django settings for Sentiment_analysis project.
+
+Generated by 'django-admin startproject' using Django 3.1.5.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.1/ref/settings/
+"""
+
+from pathlib import Path
+import os
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'g%sfzyu*_m719@hk7qh1hqg-e+!9y6hw2x5)aq1q)4t%hk!!+t'
```

Comment:
replace secret with: `not for production`

It is a bad practice to store secrets in the public repository.  Using some dumme phrase highlights that you know it.